regparser/tree/xml_parser/reg_text.py

```diff
@@ -171,15 +180,29 @@ def build_from_section(reg_part, section_xml):
         if ch.tag == 'STARS':
             nodes.append(Node(label=[mtypes.STARS_TAG]))
         elif not markers_list:
-            section_texts.append((text, tagged_text))
+            # is this a bunch of definitions that don't have numbers next to them?
+            if subject_text.find('Definitions.') > -1:
+                if text.find('means') > -1:
+                    def_marker = text.split('means')[0].strip().split()
+                    def_marker = ''.join([word[0].upper() + word[1:] for word in def_marker])
+                else:
+                    def_marker = 'def{0}'.format(i)
+                n = Node(text, label=[def_marker], source_xml=ch)
+                n.tagged_text = tagged_text
+                n.defn_no_number = True
+                nodes[-1].children.append(n)
+            else:
+                section_texts.append((text, tagged_text))
```

Comment:
This is a valiant effort. I'm not too confident that the rest of the system will accommodate those type of node labels, so keep your eyes open. In particular, I vaguely recall us needing to special case any node with a label ending with `p#` or `h#`, but I don't remember why. @theresaanna maybe you do?

There's existing functionality to pull out definitions which it'd probably make sense to reuse (even though the interface isn't pretty). See how the terms layer is called here: https://github.com/cfpb/regulations-parser/blob/master/regparser/layer/key_terms.py#L43-47 (that's code to exclude definitions that look like key terms). Also, note that the definitions it returns may include spaces.